store/storetest/team_store.go

```diff
@@ -770,54 +771,58 @@ func testTeamMembers(t *testing.T, ss store.Store) {
 	m2 := &model.TeamMember{TeamId: teamId1, UserId: model.NewId()}
 	m3 := &model.TeamMember{TeamId: teamId2, UserId: model.NewId()}
 
-	if r1 := <-ss.Team().SaveMember(m1, -1); r1.Err != nil {
-		t.Fatal(r1.Err)
+	if _, err := ss.Team().SaveMember(m1, -1); err != nil {
+		t.Fatal(err)
 	}
 
-	store.Must(ss.Team().SaveMember(m2, -1))
-	store.Must(ss.Team().SaveMember(m3, -1))
+	_, err := ss.Team().SaveMember(m2, -1)
+	require.Nil(t, err)
+	_, err = ss.Team().SaveMember(m3, -1)
+	require.Nil(t, err)
 
-	if ms, err := ss.Team().GetMembers(teamId1, 0, 100, nil); err != nil {
+	var ms []*model.TeamMember
```

Comment:
Why has this been added and the assignments changed below?

Reply:
same shadowing variable problem, but with your next comment it gets simplified.

Reply:
Fixed